server/models/UserModel.js

```diff
@@ -0,0 +1,21 @@
+const mongoose = require('mongoose');
+
+const UserSchema = mongoose.Schema({
+  name: {
+    type: String,
+    required: [true, 'The name is required!'],
+    maxlength: 100,
+  },
+  email: {
+    type: String,
+    required: [true, 'The email is required!'],
+    unique: true,
+  },
+  password: {
+    type: String,
+    required: [true, 'The password is required!'],
+    minlength: 6,
```

Comment:
I don't believe this is needed because the hash will always be greater than 6 characters, you could probably move this validation to the same pre save function I referred to above

Reply:
Removed the minlength.